src/bsoncxx/lib/bsoncxx/v1/config/config.hpp.in

```diff
@@ -12,27 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#if !defined(BSONCXX_V1_CONFIG_CONFIG_HPP)
+#define BSONCXX_V1_CONFIG_CONFIG_HPP
+
 #cmakedefine BSONCXX_POLY_USE_IMPLS
 #cmakedefine BSONCXX_POLY_USE_STD
 
 ///
 /// @file
 /// Provides macros describing the bsoncxx library configuration.
 ///
-/// @warning This header is not standalone includable!
```

Comment:
This warning is no longer applicable due to the v1 macro guard strategy and backward compatibility with the v_noabi include-via-prelude pattern. Users are now free to include `config.hpp` and `version.hpp` directly (even via `v_noabi` headers).